### PR TITLE
Create Printpattern.py

### PR DESCRIPTION
**Pull request by kunjikatripathi**
added printing program

### CODE REVIEW

File: Printpattern.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+row = int(input("How many line You wish to print"))
+pattern = int(input("Input 1 for Asending and 0 for Desending"))
+if pattern <= 1:
+    n = bool(pattern)
+    print(pattern)
+    if n == True:
+        for i in range(1,row+1):
+            print("\n",end=" ")
+            for j in range(1,i+1):
+                print("*",end=" ")
```
Comment (ashishaaashu) [inline Printpattern.py:19]: if n == False:
         for i in range(row,0,-1):
            print("\n",end=" ")
            for j in range(1,i+1):
                print("*", end=" ")
else:
    print("Invalid!!")

Comment (ashishaaashu) [reply Printpattern.py:19]: missing code from ur program

**Review by ashishaaashu — CHANGES_REQUESTED** [review]

**Comment by kunjikatripathi** [comment]
Yup!! that's missing . thankx

**Comment by kunjikatripathi** [comment]
#https://www.facebook.com/100005198139377/posts/1502511503265440/
#subscribe by code house
 #codehouse merged this pull request also

**Comment by kunjikatripathi** [comment]
#https://www.facebook.com/100005198139377/posts/1502511503265440/
#subscribe by code house
#codehouse merged this pull request also

**Comment by codehouseindia** [comment]
Merged

**Pull request merged**